### PR TITLE
Addressed issue #988

### PR DESCRIPTION
**Pull request by kydos**
zenoh [https://crates.io/crates/zenoh] is an innovative communication framework entirely written in async Rust. We focus on the unification of data in motion and data at rest as such zenoh can be used as a pub/sub or a geo-distributed storage system.

This is the pull request that adds the suggested changes.

Thanks!

### CODE REVIEW

**Comment by luciusmagn** [comment]
Can you add branch to the github action badges?

File: README.md [hunk 1]
```diff
@@ -190,7 +190,6 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 * [TerrabaseDB](https://github.com/terrabasedb/terrabasedb) — A multi-model NoSQL database ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/terrabasedb/terrabasedb/Tests?style=flat-square)
 * [tikv](https://github.com/tikv/tikv) — A distributed KV database in Rust [![build badge](https://circleci.com/gh/tikv/tikv.svg?style=shield&circle-token=36bab0a8e43edb0941b31c38557d2d9d0d58f708)](https://app.circleci.com/pipelines/github/tikv/tikv)
 * [sled](https://crates.io/crates/sled) — A (beta) modern embedded database [![Build Status](https://github.com/spacejam/sled/workflows/Rust/badge.svg?branch=master)](https://github.com/spacejam/sled/actions?workflow=Rust)
-
```
Comment (palfrey) [inline README.md:193]: Blank line needs re-adding

**Comment by palfrey** [comment]
This has conflicts with current master README. Can you fix them please?

**Comment by palfrey** [comment]
Closing for the moment while we await charges for the issues above

**Pull request closed without merging**